File: documentation/user/en/use/api/example/fetching-example.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					and(
						entityPrimaryKeyInSet(1),
						entityLocaleEquals(Locale.forLanguageTag("en")),
						priceInPriceLists("basic"),
						priceInCurrency(Currency.getInstance("CZK"))
					)
				),
				require(
					entityFetch(
						attributeContent("name"),
						associatedDataContentAll(),
						priceContentRespectingFilter(),
						referenceContent("brand")
					)
				)
			)
		);
	}
);

File: documentation/user/en/use/api/example/fetching-example.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		And(
        			EntityPrimaryKeyInSet(1),
        			EntityLocaleEquals(CultureInfo.GetCultureInfo("en")),
        			PriceInPriceLists("basic"),
        			PriceInCurrency(new Currency("CZK"))
        		)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("name"),
        			AssociatedDataContentAll(),
        			PriceContentRespectingFilter(),
        			ReferenceContent("brand")
        		)
        	)
        )
	)
);